README.rst

```diff
@@ -67,11 +60,11 @@ KairosDB frontend, i.e. for manually query of metrics:
 Issues
 ------
 
-* every LDAP user has admin role (because I could not get the OpenLDAP "memberOf" overlay to work
+* Every LDAP user has an admin role, because we could not get the OpenLDAP "memberOf" overlay to work.
 
-* If single containers do not start up ssh into the vagrant box and run the start.sh script again manually or use the start-services.sh script to restart single components. Later one takes parameters like controller or worker.
+* If single containers do not start up SSH into the Vagrant box and run the start.sh script again manually, or use the start-services.sh script to restart single components, later ones take parameters like controller or worker.
```

Comment:
probably a comma or "then" is missing:

"If ... do not start up then ssh into ..."